src/components/comment/comment.twig

```diff
@@ -0,0 +1,79 @@
+{% set _heading_depth = min(heading_depth|default(3), 6) %}
+
+<article class="c-comment{% if comment.children is not empty %} c-comment--thread{% endif %}" id="comment-{{comment.ID}}">
+  <header class="c-comment__header">
+    <h{{_heading_depth}} class="c-comment__heading">
+      {{comment.author.name}}
+      <span class="u-hidden-visually">
+        {% if comment.is_child %}replied{% else %}said{% endif %}:
+      </span>
+    </h{{_heading_depth}}>
+  </header>
+  <div class="c-comment__object">
+    {% include '@cloudfour/components/avatar/avatar.twig' with {
+      src: comment.avatar,
+      size: 'full'
+    } only %}
+  </div>
+  {% embed '@cloudfour/objects/rhythm/rhythm.twig' with {
+    class: 'c-comment__content o-rhythm--condensed'
+  } %}
+    {% block content %}
+      {{comment.comment_content|raw}}
+    {% endblock %}
+  {% endembed %}
+  <footer class="c-comment__footer">
+    <div class="c-comment__meta">
+      <div class="c-comment__meta-detail">
+        <a class="c-comment__meta-link"
+          href="{{comment.link}}"
+          title="Permalink to {{comment.author.name}}’s {{comment.date|date('M j')}} {% if comment.is_child %}reply{% else %}comment{% endif %}">
+          <span class="c-comment__meta-icon">
+            {% include '@cloudfour/components/icon/icon.twig' with {
+              name: 'link',
+              inline: true,
+              aria_hidden: 'true',
+            } only %}
+          </span>
+          <time datetime="{{comment.date|date('Y-m-d')}}">
+            {{comment.date|date('M j, Y')}}
+          </time>
+        </a>
+      </div>
+      {% if source %}
+        <div class="c-comment__meta-detail">
+          via <a class="c-comment__meta-link" href="{{source.url}}">{{source.name}}</a>
+        </div>
+      {% endif %}
+      {% if demo_control %}
+        <div class="c-comment__meta-control">
+          {% include '@cloudfour/components/button/button.twig' with {
+            type: 'button',
+            class: 'c-button--tertiary',
+            content_start_icon: 'speech-balloon',
+            label: 'Reply'
+          } only %}
+        </div>
+      {% endif %}
+    </div>
+    {% if comment.children is not empty %}
+      {% set _section_heading_depth = min(_heading_depth + 1, 6) %}
+      {% set _child_heading_depth = min(_section_heading_depth + 1, 6) %}
+      <h{{_section_heading_depth}} class="u-hidden-visually">
+        Replies
+      </h{{_section_heading_depth}}>
```

Comment:
Clever! I totally missed this the first time I turned VoiceOver on.

For a moment, I had a slight concern that seeing multiple "Replies" headings might be confusing within the context of the VO rotor but since the rotor lists them in order, I think there is enough contextual information that it makes sense. 👍 

![IMG_9939](https://user-images.githubusercontent.com/459757/124841280-b788db80-df41-11eb-870f-c68f35f61d68.jpg)

Reply:
@gerardo-rodriguez I've made a slight adjustment based on this observation, adding the original comment author's name to the "Replies" heading. I think this makes the headings more distinct and the relationship a bit more clear between heading levels now.

![IMG_4292](https://user-images.githubusercontent.com/69633/124953110-aedbe800-dfc9-11eb-9cc7-635da1f62d32.jpg)

Reply:
This seems like a great improvement :+1: